### ISSUE TITLE
CF Workers - SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by xar** [nuxt/framework#4226]
### Environment

Nuxt CLI v3.0.0-27490192.4253d2e                                                                                                                                                                      
Nuxt project info:                                                                                                                                                     

------------------------------
- Operating System: `Darwin`
- Node Version:     `v16.13.0`
- Nuxt Version:     `3.0.0-27490192.4253d2e`
- Package Manager:  `npm@8.1.0`
- Builder:          `vite`
- User Config:      `nitro`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------

### Reproduction

Install latest nuxt, run `NITRO_PRESET=cloudflare nuxi build`, use `wrangler dev` or miniflare or check `server/index.mjs`

### Describe the bug

After upgrading from `3.0.0-27484505.9f6eff7`, building with `NITRO_PRESET=cloudflare nuxi build` and running miniflare I get exception `SyntaxError: Cannot use import statement outside a module` and unable to start / deploy. Builds in previous versions ended with `nuxi preview` and bundle was nicely compiled. But now when I open `index.mjs` in server directory It's in es6 which is probably not supported...

**Comment by phlqp** [comment]
Try to run local: **node .output/server/index.mjs**
Results:
node:internal/errors:464
    ErrorCaptureStackTrace(err);
Error [ERR_MODULE_NOT_FOUND]: Cannot find package '_process' imported from ../.output/server/node_modules/unenv/runtime/polyfill/process.mjs
    at new NodeError (node:internal/errors:371:5)
    at packageResolve (node:internal/modules/esm/resolve:932:9)
    at moduleResolve (node:internal/modules/esm/resolve:978:18)
    at defaultResolve (node:internal/modules/esm/resolve:1080:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:530:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:251:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:79:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'

**Comment by larswww** [comment]
same here

**Comment by jontybrook** [comment]
Also experiencing this with nuxt v3.0.0-27491748.3186841

**Comment by pi0** [comment]
Sorry for the regression. It should be resolved with `nitropack@0.2.2` (2e5de6cf37f4954cbd1492212f4537c44ab681e4). You can get the fix using `npx nuxi upgrade --force`.

**Comment by DaniDantur** [comment]
The same is happening to us with nuxt-edge. We tried everything we found online but nothing improved our situation (sorry for not providing a list of what we tried, it was a mess lately since we have been stuck for almost a month now and we are kind of desperate). Any help will be appreciated.

Node v16.14.0
yarn v1.22.17

From yarn list:
nitropack@0.2.11
nuxt-edge@2.16.0-27358576.777a4b7f
nuxt/bridge@3.0.0-27498038.f18e68b
├─ nitropack@^0.2.7
nuxt/cli-edge@2.16.0-27358576.777a4b7f

When `NITRO_PRESET=cloudflare yarn build`:
Nuxt CLI v3.0.0-rc.2-27529732.ebc27ce
